packages/jest-haste-map/src/lib/__tests__/docblock-test.js

```diff
@@ -16,6 +16,17 @@ const docblock = require('../docblock');
 
 describe('docblock', () => {
 
+  it('extracts valid docblock with line comment', () => {
+    const code =
+      '/**' + os.EOL + ' * @providesModule foo' + os.EOL +
+      '* // TODO: test' + os.EOL + '*/' + os.EOL +
```

Comment:
At FB we actually have this:

```
/**
 * @requires A
 *                   B
 */
```

and similar. So definitely a feature but our docblock parsers in JS never had to support this I assume.